cost-analyzer/values.yaml

```diff
@@ -3272,6 +3272,11 @@ costEventsAudit:
 #     key: ""
 #     secretname: productkeysecret # Reference an existing k8s secret created from a file named productkey.json of format { "key": "enterprise-key-here" }. If the secretname is specified, a configmap with the key will not be created.
 #     mountPath: "/some/custom/path/productkey.json" # (use instead of secretname) Declare the path at which the product key file is mounted (eg. by a secrets provisioner). The file must be of format { "key": "enterprise-key-here" }.
+#   smtp:
+#     enabled: false
+#     config: ""
+#     secretname: smtpconfigsecret
+#     mountPath: "/some/custom/path/smtp.json"
```

Comment:
Why do we need a variable mount path?

Reply:
For the entirety of this PR, I followed the example set by `productkey`. Based on the comments laid out in `productkey`, variable mount paths are used to enable secrets provisioners.

Now, I'd rate my personal experience with secrets provisioners at around zero out of ten, but given the context of what we're working with, I'd say that's possibly somewhat reasonable.

Reply:
Ah, same with the experience. I'll trust those comments! Thanks for the explanation

Reply:
Let's also add a comment above this to clearly explain what this block does (enables use of a custom SMTP server which trumps Kubecost's built-in, external SMTP server).

Reply:
And what's in `config`?

Reply:
@chipzoller Very valid points; I'll make that change and clarify what should go in `config`.